lighthouse-core/test/gather/driver-test.js

```diff
@@ -262,6 +311,7 @@ describe('.gotoURL', () => {
       id: 'ABC', loaderId: '', securityOrigin: '', mimeType: 'text/html', domainAndRegistry: '',
       secureContextType: /** @type {'Secure'} */ ('Secure'),
       crossOriginIsolatedContextType: /** @type {'Isolated'} */ ('Isolated'),
+      gatedAPIFeatures: [],
```

Comment:
what necessitated this?

Reply:
I was getting some TS errors when running `yarn type-check` on my end. I can remove to see if it's necessary for CI.

Reply:
> I was getting some TS errors when running `yarn type-check` on my end. I can remove to see if it's necessary for CI.

ah, yeah, I was wondering because we don't use it anywhere, but looks like it's a new property on `Page.Frame` and we need this to type check one.